internal/oauth2/lazy_token_test.go

```diff
@@ -54,19 +62,21 @@ func TestLazyToken(t *testing.T) {
 	require.Equal(t, "mock-token-1", token)
 
 	// ensure no token is not part of the logs
-	assert.Equal(t, 1, len(hook.Entries))
-	assert.Contains(t, hook.Entries[0].Message, "Acquired new token")
-	assert.NotContains(t, hook.Entries[0].Message, "mock-token-1")
-	hook.Reset()
+	field := collector.Last()["msg"]
+	assert.Equal(t, 1, len(collector.All()))
+	assert.Contains(t, field, "Acquired new token")
+	assert.NotContains(t, field, "mock-token-1")
+	collector.Reset()
```

Comment:
This test is probably unnecessary, but I did not want to remove any code in this patch, so I created similar helper handler (aka hook) to do the same logic.